docs/_docs/auth.md

```diff
@@ -69,25 +71,33 @@ _→ [Add Digest Auth to your Zapier Integration](https://platform.zapier.com/do
+Zapier always includes the app's name in each account label. You can add:
 
 - Plain text that will be included after your app's full name
-- Input fields with data users entered in the authentication form, referenced with {% raw %}`{{bundle.authData.field}}`{% endraw %} variables, replacing `field` for your input field name. With Basic auth, use `username` as the field.
-- Output fields from your app's authentication test API call, referenced with {% raw %}`{{bundle.inputData.field}}`{% endraw %} variables, replacing `field` for your API output field name.
+- Data that users entered in the authentication form.
+- Output fields from your app's authentication test API call.
+
+To add a field value to the connection label, enter the name of the field, surrounded by double curly braces, in the _Connection Label_ field.
+
+For example, if your auth configuration includes a `username` field, your connection label could look like:
+
+{% raw %}`{{username}}`{% endraw %}
 
-To add a connection label using an input field, enter the following in your _Connection Label_ field, optionally replacing `username` with the key for another input field your authentication form includes:
+Alternately, you can use an output field from your test API call in the Connection Label. If you aren't sure what fields will be returned, skip this step for now and go to Step 3, _Test your Authentication_.
 
-{% raw %}`{{bundle.authData.username}}`{% endraw %}
+After connecting an account, check the _Response_ tab to see the fields returned from the test request. Then, return to Step 2, and enter that field key in the _Connection Label_ field. So if the field you want to use is called `team`, enter:
 
-Alternately, you can use an output field from your test API call in the Connection Label if you know the name of the field you wish to use that your API sends to Zapier. Otherwise, skip adding the connection label first, and instead click _Connect an Account_ under step 2. Then click _Test Connected Account_ and check the _Response_ tab for a field that includes the data your app should use in the connection label. Finally, enter that field key in the following text instead of `field`, and add it to your _Connection Label_ field:
+{% raw %}`{{team}}`{% endraw %}
 
-{% raw %}`{{bundle.inputData.username}}`{% endraw %}
+When setting up a simple text connection label with dynamic fields, field keys from the authentication form and the test request are both pulled up to the top level, which is why simple field keys like `username` will work.
+ 
+You can also refer to the fields with {% raw %}`{{bundle.authData.field}}`{% endraw %} for input fields from the authentication form, replacing `field` with your input field name. Use {% raw %}`{{bundle.inputData.field}}`{% endraw %} for fields returned from the test request, replacing `field` with the field name from the API response.
```

Comment:
> field keys from the authentication form and the test request are both pulled up to the top level

This is a bit inaccurate. A simple field key `{{abc}}` only works when `abc` is stored in the connected account. Since we store all the input fields from the authentication form, all the input fields in the authentication form should be accessible with `{{abc}}`.

However, for auth types that don't have an authentication form, such as OAuth, we store "computed fields" like `access_token` and any other fields returned from the `getAccessToken` method. Those computed fields are also stored, so you can access them with `{{abc}}` as well.

If `{{abc}}` is not stored and only appears in the test API response, you should use `{{bundle.inputData.abc}}`.

Suggested change:

```suggestion
When setting up a simple text connection label with dynamic fields, field keys from the authentication form pulled up to the top level, so `{{username}}` will work. It's equivalent  to `{{bundle.authData.username}}`.
 
For fields returned from the test request, you can refer to them with `{{bundle.inputData.field}}`, replacing `field` with the field name from the API response.
```

Reply:
Same here as above: https://github.com/zapier/visual-builder/pull/327/files#r949594191

Reply:
Replied above https://github.com/zapier/visual-builder/pull/327#discussion_r949766174